app/src/main/java/org/wikipedia/theme/ThemeFittingRoomActivity.kt

```diff
@@ -0,0 +1,45 @@
+package org.wikipedia.theme
+
+import android.R
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.core.content.ContextCompat
+import org.wikipedia.Constants
+import org.wikipedia.activity.SingleFragmentActivity
+import org.wikipedia.page.ExclusiveBottomSheetPresenter
+
+class ThemeFittingRoomActivity : SingleFragmentActivity<ThemeFittingRoomFragment>(), ThemeChooserDialog.Callback {
+    private var themeChooserDialog: ThemeChooserDialog? = null
+    private val bottomSheetPresenter = ExclusiveBottomSheetPresenter()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (savedInstanceState == null) {
+            themeChooserDialog = ThemeChooserDialog.newInstance(Constants.InvokeSource.SETTINGS)
+            bottomSheetPresenter.show(supportFragmentManager, themeChooserDialog!!)
+        }
+
+        // Don't let changed theme affects the status bar color and navigation bar color
+        setStatusBarColor(ContextCompat.getColor(this, R.color.black))
+        setNavigationBarColor(ContextCompat.getColor(this, R.color.black))
```

Comment:
Can we use `android.R.color.black` here instead of importing `import android.R` to prevent IDE warning?